app/actions_test.go

```diff
@@ -5,18 +5,17 @@
 package app
 
 import (
-	"errors"
-
 	"github.com/globalsign/mgo/bson"
+	"github.com/pkg/errors"
 	"github.com/tsuru/config"
 	"github.com/tsuru/tsuru/action"
 	"github.com/tsuru/tsuru/app/bind"
 	"github.com/tsuru/tsuru/auth"
-	"github.com/tsuru/tsuru/quota"
 	"github.com/tsuru/tsuru/repository"
 	"github.com/tsuru/tsuru/router/routertest"
 	appTypes "github.com/tsuru/tsuru/types/app"
-	"gopkg.in/check.v1"
+	authTypes "github.com/tsuru/tsuru/types/auth"
+	check "gopkg.in/check.v1"
```

Comment:
You don't need to set the import name here, `check` is the default.

Reply:
I need double `check` with the Go automatic imports.